app-backend/app/src/main/resources/application.conf

```diff
@@ -66,5 +66,11 @@ featureFlags {
       name = "Profile Organization Edit",
       description = "Join / Create / Edit / Leave / Delete organizations in user profile settings"
     }
+    {
+      key = "show-make-source-histogram"
```

Comment:
I'd probably just make the flag "histogram-sync" or "source-histogram". IE reference what the actual feature is, not what the flag affects